### PR TITLE
[TASK-6795] feat(request): allow attachment download on receipt

### PR DESCRIPTION
**Pull request by jjramirezn**

### CODE REVIEW

File: src/components/Request/Pay/Views/GeneralViews/AlreadyPaid.view.tsx [hunk 1]
```diff
@@ -7,22 +7,45 @@ import * as utils from '@/utils'
...
+                            <a
+                                href={requestLinkData!.attachmentUrl}
+                                download
+                                target="_blank"
+                                className="flex w-full cursor-pointer flex-row items-center justify-center gap-1 text-h9 font-normal text-gray-1 underline "
+                            >
+                                <Icon name={'download'} />
+                                Download attachment
+                            </a>
+                        )}
```
[the remark this answers is not in the document]
Comment (Hugo0) [reply src/components/Request/Pay/Views/GeneralViews/AlreadyPaid.view.tsx]: > Add rel="noopener noreferrer" to attachment download links

Why is this necessary or a good security consideration?

File: src/components/Request/Pay/Views/GeneralViews/AlreadyPaid.view.tsx [hunk 2]
```diff
@@ -7,22 +7,45 @@ import * as utils from '@/utils'
...
     const chainAvailable: boolean = !!requestLinkData?.chainId
     const recipientAddressAvailable: boolean = !!requestLinkData?.recipientAddress
-    const dataAvailable: boolean = tokenSymbolAvailable || tokenAmountAvailable || chainAvailable || recipientAddressAvailable
-    
+    const dataAvailable: boolean =
+        tokenSymbolAvailable || tokenAmountAvailable || chainAvailable || recipientAddressAvailable
+
     return (
         <div className="flex w-full flex-col items-center justify-center gap-6 py-2 pb-20 text-center">
+            {(requestLinkData?.reference || requestLinkData?.attachmentUrl) && (
```
Comment (Hugo0) [inline src/components/Request/Pay/Views/GeneralViews/AlreadyPaid.view.tsx]: >  {(requestLinkData?.reference || requestLinkData?.attachmentUrl) && (
Comment

Q: Why do this? is it necessary? We already check for existence of both of  these individually later

**Review by Hugo0 — APPROVED** [review]

**Comment by Hugo0** [comment]
Q: I don't understand why this is necessary - didn't we already have this functionality before? When did it disappear? or did it only exist for claimLinks?

Code suggestion: If we already have it for claimlinks, maybe it would make sense to standardize into one common reference/attachment component?

**Comment by jjramirezn** [comment]
> Q: I don't understand why this is necessary - didn't we already have this functionality before? When did it disappear? or did it only exist for claimLinks?
> 
> Code suggestion: If we already have it for claimlinks, maybe it would make sense to standardize into one common reference/attachment component?

We did not have this functionality, added new commit to standarize

**Pull request merged**